### DISCUSSION TITLE
My student ID Card Keeps getting rejected with reason, Your image contains keywords indicating that it may be an acceptance letter.

### DISCUSSION THREAD

**Discussion by irajankumar** [community/community#141826] [GitHub Education]
### Hi everyone,

Hi, I'm currently perusing EPGP MBA degree program from IIM Kozhikode, India and trying to enroll into GitHub education program as student using my college email id.

Upon submission of photo of my student identity card which is issued from the institute, my application is getting rejected with the reason, "Your image contains keywords indicating that it may be an acceptance letter".

Can someone please help ? What other documents do I need to submit for application.

Regards,
Rajan Kumar

**Comment by Unmesh100** [comment]
I understand you're having trouble enrolling in the GitHub Education program with your IIM Kozhikode EPGP MBA student credentials. Here are some specific suggestions to help get your application approved:

Alternative Documentation Options:

Current enrollment verification letter from IIM Kozhikode
Screenshot of your official student portal showing current enrollment status
Official class schedule or registration document
Recent dated tuition receipt
Official transcript (with current date)


Tips for Document Submission:

Ensure the document clearly shows:

Your name
The institution name (IIM Kozhikode)
Current date/semester
Your student status


Make sure the image is clear and all text is readable
Crop the image to focus on the relevant information


If your application gets rejected again:

Contact GitHub Education support directly at [education@github.com](mailto:education@github.com)
Explain that you're an EPGP MBA student at IIM Kozhikode
Mention the specific rejection reason you received
Offer to provide additional verification if needed.
